other/preact.package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
What's this for?

Reply:
`package.json` file needs to be present in `preact` folder since node.js looks into it to figure out which file to import. This file is used as a blueprint for it. Perhaps I could use main package.json file for this as well :thinking: